src/Html/Builder.php

```diff
@@ -418,7 +419,14 @@ public function addCheckbox(array $attributes = [])
             'printable'      => true,
             'width'          => '10px',
         ], $attributes);
-        $this->collection->push(new Column($attributes));
+        $column = new Column($attributes);
```

Comment:
Maybe add an empty line after `$column = new Column($attributes);`, StyleCI most likely doesn't like it like this. 👍